CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.6.0 (2020-10-15)
+
+- Adjusted various datepicker component calendar button sizes for the modern theme. [#235](https://github.com/blackbaud/skyux-datetime/pull/235)
```

Comment:
I think we usually stick to a small handful of opening verbs and that we usually use `Updated` rather than `Adjusted`. 

Also, `various datepicker component calendar button sizes` is a little tricky to digest at first blush, and I think that it would clearer if we rearranged it a little to `calendar button sizes for the datepicker component`.

That said, neither of these suggestions is a big deal, so I'm going to approve the PR. If you decide to make the changes and need me to re-approve, just let me know.